common/src/main/java/com/wynntils/services/mapdata/providers/type/MapDataOverrideProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © Wynntils 2024-2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.mapdata.providers.type;
+
+import com.wynntils.services.mapdata.attributes.type.MapAttributes;
+import com.wynntils.services.mapdata.features.type.MapFeature;
+import com.wynntils.services.mapdata.type.MapDataProvidedType;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+public interface MapDataOverrideProvider {
+    /**
+     * Get the attributes that are overriden by this provider.
+     *
+     * @param mapFeature The map feature to get the attributes for.
+     * @return The attributes that are overriden by this provider.
+     */
+    MapAttributes getOverrideAttributes(MapFeature mapFeature);
+
+    /**
+     * Get the feature ids that are overriden by this provider.
```

Comment:
The wording is similar to getOverrideAttributes, but I think the meaning is rather different: This is the list of feature id:s that this provider does serve overridden attributes for.

I'm not sure how we should phrase this, but the getOverrideAttributes should not share description with getOverridden[Feature|Category]Ids.

Reply:
Okay, I've changed this.